GoogleForCreators/web-stories-wp
# Add missing date field to fixture

## Summary
The API response for getting a story in fixture for Karma tests did not include the `date` field that we're using in the editor. This PR adds the date field, too. 


## Relevant Technical Choices



## To-do

- [x]  Wait until the tests finish running.



## User-facing changes



## Testing Instructions
Review the Percy changes and ensure that "Immediately" is displayed in the Publish time where relevant.


### QA






This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #
